test/support/coronavirus_helpers.rb

```diff
@@ -0,0 +1,130 @@
+module CoronavirusHelpers
```

Comment:
There's two lines of change on this file when compared to the helper it's replacing. I'm going to assume that that's just the `module CoronavirusHelpers` line 1 and the `end` that this necessitates rather than going thought line by line 😂

Reply:
Yup, that's correct no further modifications. I should have done the module change and rename as separate commits then it'd have been easier to see the changes with whitespace off.